draft-ietf-quic-recovery.md

```diff
@@ -102,7 +102,9 @@ In-flight:
 
 : Packets are considered in-flight when they have been sent
   and neither acknowledged nor declared lost, and they are not
-  ACK-only.
+  ACK-only.  For example, packets containing STREAM frames are
```

Comment:
I think that the "for example" is missing context.  Perhaps:

> : Packets are considered in-flight when they have been sent and neither acknowledged nor declared lost.  ACK-only packets are not considered in-flight, though all other packets are, including packets that contain data that would not otherwise be sent again, such as a packet containing only STREAM frames for a reset stream.

Reply:
That works for me.  @janaiyengar what do you think?